sdk/vision/ai-vision-image-analysis-rest/review/ai-vision-image-analysis.api.md

```diff
@@ -117,7 +118,7 @@ export interface AnalyzeFromUrlQueryParamProperties {
     "gender-neutral-caption"?: boolean;
     "model-version"?: string;
     "smartcrops-aspect-ratios"?: number[];
-    features: string[];
+    features: VisualFeatures[];
```

Comment:
Should this be called out as a breaking change in CHANGELOG.md?

Reply:
It actually doesn't break string users. The tests use strings and didn't require an update.

https://github.com/Azure/azure-sdk-for-js/blob/830c5c6f11f0d48bc76abc8b402cec4ef704c937/sdk/vision/ai-vision-image-analysis-rest/review/ai-vision-image-analysis.api.md?plain=1#L275

Let me see if I can conjure an easier sample though.

Reply:
And no... The generator changed the name to an alias to string, but didn't output the possible values. So customers are still going to need to look the strings up in docs.